backend/lib/factory.ts

```diff
@@ -116,10 +117,10 @@ export const getOfficesGateway = (context: ApplicationContext): OfficesGateway =
 
 export const getOfficesRepository = (context: ApplicationContext): OfficesRepository => {
   if (context.config.get('dbMock')) {
-    if (!mockOrdersRepository) {
-      mockOrdersRepository = MockMongoRepository.getInstance(context);
+    if (!mockOfficesRepository) {
```

Comment:
Note that a previous PR for CAMS-500 replaced the MockOfficesRepository with the MockMongoRepository in an attempt to let the Flexion environment use staff from the db rather than hard-coded staff. This is not currently working. If we want to revert to the previous use of the hard-coded staff, we can for now.

Reply:
Reverting to prior behavior for now.